merickson/Floe
# Fix for who flood that clutters channels when connected to IRC via ZNC.

### Issue

When connected to an IRC server via ZNC there is a periodic flood of who messages that gets dumped into the active channel window. This flooding appears to happen every 60-ish seconds which can cause monitoring an IRC channel near impossible.
### Summary

Marking `RPL_ENDOFWHO` and `RPL_WHOREPLY` messages as handled appears to clear while still letting users use the `/whois Nick` command. 

**Note** I have no idea why `Floe.UI/ChatBox/ChatPresenter_Scrolling.cs` is marked as changed. I did not touch this file and `git reset upstream/master --hard` would never clear the file... I am open to any suggestions on how to "fix" it. This patch was developed on Win 7 with VS2013 Community edition.
### Testing
- From `master` build and connect to a ZNC server running version 1.4 (assumed to work with all versions of ZNC). 
- Idle in a channel with multiple users for up to 5 minutes. This actually works better when connected to multiple channels
- Observe the who lists being output to the active channel window for all channels you are connected to.
- Apply the code from this fix and repeat the steps above. The difference will now be that who messages do not clutter your active channel window.
